packages/client/src/components/Dashboard/TeamMemberListItem.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { IconButton } from '@mui/material';
+import EditIcon from '@mui/icons-material/Edit';
+import DeleteIcon from '@mui/icons-material/Delete';
+
+const TeamMemberListItem = ({
+  member,
+  handleEditMember,
+  handleDeleteMember,
+}) => {
+  return (
+    <li key={member.member_id}>
+      <div style={{ display: 'flex', alignItems: 'center' }}>
```

Comment:
Please move this to a css class. 
None of the companies are using this syntax, since inline css is very difficult to debug & mantain.